.github/workflows/test.yml

```diff
@@ -71,6 +71,13 @@ jobs:
           output: both
           thresholds: '60 80'
 
+      - name: Add Coverage PR Comment
+        uses: marocchino/sticky-pull-request-comment@v2
+        if: github.event_name == 'pull_request'
+        with:
+          recreate: true
+          path: code-coverage-results.md
+
```

Comment:
With `codecov` we automatically have comments inside PR at the end of a CI run, so we can remove this action